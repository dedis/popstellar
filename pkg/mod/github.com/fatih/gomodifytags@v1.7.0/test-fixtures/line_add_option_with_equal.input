package foo

type foo struct {
	bar string
	t   bool `json:"t"`
	qux int  `json:"qux"`
}
