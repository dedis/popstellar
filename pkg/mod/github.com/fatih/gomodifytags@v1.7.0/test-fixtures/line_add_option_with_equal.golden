package foo

type foo struct {
	bar string `validate:"bar,max=32"`
	t   bool   `json:"t" validate:"t,max=32"`
	qux int    `json:"qux" validate:"qux,max=32"`
}
